api/src/db/migrations/v149_extract_business_data.ts

```diff
@@ -0,0 +1,861 @@
+import { v148Business, v148UserData } from "@db/migrations/v148_add_vacant_building_field";
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v148_to_v149 = (v148Data: v148UserData): v149UserData => {
+  return {
+    ...v148Data,
+    businesses: Object.fromEntries(
+      Object.values(v148Data.businesses)
+        .map((business: v148Business) => migrate_v148Business_to_v149Business(business))
+        .map((currBusiness: v149Business) => [currBusiness.id, currBusiness])
+    ),
+    version: 149,
+  } as v149UserData;
+};
+
+export const migrate_v148Business_to_v149Business = (business: v148Business): v149Business => {
+  return {
+    ...business,
+    profileData: {
+      ...business.profileData,
+      vacantPropertyOwner: undefined,
```

Comment:
pebble - why is this line included?

Reply:
I’m not sure which line you’re specifically referring to, but the primary change in this version compared to the previous one is the addition of the version field, i.e `version: 149`. This modification is part of the update being applied to the businesses object to reflect the migration changes